php/class-customize-snapshot-manager.php

```diff
@@ -225,7 +226,7 @@ public function add_snapshot_uuid_to_return_url() {
 			&&
 			$this->current_snapshot_uuid
 			&&
-			$this->is_theme_active()
+			$this->customize_manager->is_theme_active()
 			&&
```

Comment:
@westonruter Not sure why `$this->customize_manager->is_theme_active()` is returning `false` in 4.6.1 which is failing phpunit test.